PracticeGit/PracticeGit.vcxproj

```diff
@@ -20,10 +20,18 @@
   </ItemGroup>
   <ItemGroup>
     <ClCompile Include="Program.c" />
+<<<<<<< HEAD

```

Comment:
You have to remove the merge markers once you have solved the problem. Right now your project is not loading 
![image](https://github.com/user-attachments/assets/a25f00de-e7fc-48e1-bb84-4b1399cc10d8)